### PR TITLE
Clarify query parameter safety and embed rules

### PR DESCRIPTION
**Pull request by susodapop**
Address #260.

### CODE REVIEW

**Comment by susodapop** [comment]
Does this look accurate to you @rauchy?

File: src/pages/kb/user-guide/querying/query-parameters.md [hunk 1]
```diff
@@ -124,6 +124,20 @@ WHERE org_id = {{org_id}} AND created_at > '{{start_date}}'
 
 We use two parameters: `{{org_id}}` and `{{start_date}}`.
 
+**Can I use parameters in embedded visualizations?**
+
+Yes, with one exception. If a query uses a Text type parameter, it cannot be embedded because text parameters are not safe from SQL injection. All other types of query parameters can be safely embedded in visualizations.
```
Comment (arikfr) [inline src/pages/kb/user-guide/querying/query-parameters.md]: Maybe worth pointing out that dropdowns with text are fine, it's just the explicit Text parameter type that is of an issue.

Comment (arikfr) [reply src/pages/kb/user-guide/querying/query-parameters.md]: Also worth mentioning that this is true from Redash v8 or the hosted one.

File: src/pages/kb/user-guide/querying/query-parameters.md [hunk 2]
```diff
@@ -124,6 +124,20 @@ WHERE org_id = {{org_id}} AND created_at > '{{start_date}}'
...
+
+**Can I use parameters in embedded dashboards?**
```
Comment (rauchy) [inline src/pages/kb/user-guide/querying/query-parameters.md]:
```suggestion
**Can I use parameters in shared dashboards?**
```

File: src/pages/kb/user-guide/querying/query-parameters.md (hunk 1 above)
Comment (rauchy) [reply src/pages/kb/user-guide/querying/query-parameters.md]: > Maybe worth pointing out that dropdowns with text are fine, it's just the explicit Text parameter type that is of an issue.

I feel that @susodapop's phrasing is pretty explicit.

Comment (arikfr) [reply src/pages/kb/user-guide/querying/query-parameters.md]: And here's why I thought it's worth making it explicit:

image

I would consider going as far as making a table with all parameter types and a column marking if they are safe or not.

@susodapop @rauchy @ranbena

Comment (susodapop) [reply src/pages/kb/user-guide/querying/query-parameters.md]: I'll adjust this and open a fresh PR

**Comment by susodapop** [comment]
Since this PR is closed, picked up with #274.

**Pull request merged**